lib/src/main/java/org/team100/lib/util/CameraAngles.java

```diff
@@ -52,61 +33,56 @@ public CameraAngles(
      * and sim
      */
     public CameraAngles() {
-        m_downwardAngleDegrees = 0;
-        m_horzFOVDegrees = 0;
-        m_vertFOVDegrees = 0;
-        m_horzResolution = 0;
-        m_vertResolution = 0;
+        m_downwardAngleRads = 0;
         m_cameraHeightMeters = 0;
         m_xOffset = 0;
         m_yOffset = 0;
     }
 
     /**
+     *         0 should be center of FOV
      * @return A robot relative translational x value of an object in a camera in
      *         meters
      */
-    public double getX(double vertPixels) {
+    public double getX(double vertFOVRads) {
         double x = -1.0 * m_cameraHeightMeters
-                * Math.tan(vertPixels * (Math.toRadians(m_vertFOVDegrees / m_vertResolution))
-                        + Math.toRadians(90 - m_vertFOVDegrees / 2 - m_downwardAngleDegrees));
+                * Math.tan(vertFOVRads + Math.PI/2 - m_downwardAngleRads);
         return x - m_xOffset;
     }
 
     /**
+     *         0 should be center of FOV 
      * @return A robot relative translational y value of an object in a camera in
      *         meters
      */
-    public double getY(double horzPixels, double vertPixels) {
-        double x = getX(vertPixels);
-        double y = -1.0 * x
-                * Math.tan(Math.toRadians(m_horzFOVDegrees) * (horzPixels
-                        - m_horzResolution / 2) / m_horzResolution);
+    public double getY(double vertFOVRads, double horzFOVRads) {
```

Comment:
same issue with the word "FOV" here.  i'm pretty sure you're trying to represent the same concept that WPILib uses Rotation3d for, so maybe you should just do that?

also it would be good for x and y not to be two separate public methods.  make one public method that takes a camera-coordinate thing (perhaps Rotation3d) and returns a robot-coordinate thing (perhaps Translation2d)